zun_kata.md

```diff
@@ -0,0 +1,117 @@
+# Introduction
+
+This guide describes how to get Kata containers to work with OpenStack Zun using DevStack on Ubuntu 16.04. Running DevStack with this guide will setup Docker and Clear Containers 2.0, which you replace with Kata Containers. Currently, the instructions are based on the following links:  
+https://docs.openstack.org/zun/latest/contributor/quickstart.html
+
+https://docs.openstack.org/zun/latest/admin/clear-containers.html
+
+https://github.com/kata-containers/documentation/blob/master/install/ubuntu-installation-guide.md
```

Comment:
I would put a dash `-` in front of those links to get them sorted as bullet points.